src/buffer/out/textBuffer.cpp

```diff
@@ -2207,3 +2208,34 @@ HRESULT TextBuffer::Reflow(TextBuffer& oldBuffer,
 
     return hr;
 }
+
+// Method Description:
+// - Adds a hyperlink to our hyperlink table
+// Arguments:
+// - The hyperlink URI
+void TextBuffer::AddHyperlinkToMap(std::wstring uri) noexcept
+{
+    _hyperlinkMap.insert(std::pair<SHORT, std::wstring>(_curHyperlinkId, uri));
+    _curHyperlinkId++;

```

Comment:
nit: prefix increment is marginally better as you don't need the return value that postfix gives you.

Reply:
Fixed!